.github/workflows/cicd-main.yml

```diff
@@ -3887,6 +3887,38 @@ jobs:
         rm -rf tests/collections/llm/gpt_pretrain_results
         rm -rf tests/collections/llm/gpt_index_mappings
 
+  L2_NeMo_2_llama3_pretraining_recipe:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_llama3_pretraining_recipe') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+
+        python tests/collections/llm/llama3_pretraining.py \
+        --seq-length 1024 \
+        --devices=2 \
+        --max-steps=6 \
+        --early-stop=3 \
+        --experiment-dir=tests/collections/llm/llama_pretrain_results \
```

Comment:
we started - by convention - using /tmp as output directory
AFAIU this is output of the test, so could you update to use /tmp instead